src/types.ts

```diff
@@ -0,0 +1,12 @@
+import StyleDictionary, { Config } from 'style-dictionary'
+
+export type BrandsConfig = {
+  [key: string]: {
+    platforms: string[]
+    themes: string[]
+  }
+}
+
+export type FormatterConfig = {
+  [key: string]: (sd: typeof StyleDictionary, brand: string, theme: string) => Config
```

Comment:
je recommande d'extraire le type de la fonction pour pouvoir la réutiliser facilement

```suggestion
export type Formatter = (sd: typeof StyleDictionary, brand: string, theme: string) => Config

export type FormatterConfig = {
  [key: string]: Formatter
```